view/library/browse.php

```diff
@@ -75,8 +78,11 @@
 					<?php
 					$sendToEditorUrl = KalturaHelpers::generateTabUrl( array( 'tab' => 'kaltura_browse', 'kaction' => 'sendtoeditor', 'entryIds' => array( $mediaEntry->id ) ) );
 					?>
-
-					<div id="entryId_<?php echo esc_attr( $mediaEntry->id ); ?>" class="showName" title="<?php echo esc_attr( 'Click to edit' ); ?>">
+					<?php
+					$canUserEditMedia = $kalturaModel->canLoggedInUserEditMedia($mediaEntry);
```

Comment:
Please take a look and let me know if this solution is OK.
I am not sure whether it's smart to add properties to the entry object, but I couldn't think of a simpler solution...